HelpMyStreetFE/HelpMyStreetFE/Services/Requests/FilterService.cs

```diff
@@ -30,13 +39,86 @@ public SortAndFilterSet GetDefaultSortAndFilterSet(JobSet jobSet, JobStatuses? j
+                    new FilterField<PartOfDay>() {Value = PartOfDay.Afternoon, Label = "Afternoon Shifts", IsSelected = true},
+                    new FilterField<PartOfDay>() {Value = PartOfDay.Night, Label = "Night Shifts", IsSelected = true}
+                },
+                OrderBy = new List<OrderByField>
+                {
+                    new OrderByField() {Value = OrderBy.DateDue_Ascending, Label = "Soonest"},
+                    new OrderByField() {Value = OrderBy.DateDue_Descending, Label = "Furthest Away"},
+                    
+                },
+                DueInNextXDays = new List<FilterField<int>>
+                    {
+                        new FilterField<int> { Value = 1, Label = "Today" },
+                        new FilterField<int> { Value = 7, Label = "This week" },
+                        new FilterField<int> { Value = 14, Label = "Next 2 weeks" },
+                        new FilterField<int> { Value = 999, Label = "Show all", IsSelected = true }
+                    },
+
+            };
+
+
+            return filterSet;
+        }
+
+        private async Task<SortAndFilterSet> GetGroupShiftsFilterSet(User user, JobStatuses? jobStatus)
+        {
+            var filterSet = await GetShiftsFilterSet(user);
+
+            filterSet.OrderBy.Append(new OrderByField() { Value = OrderBy.Emptiest, Label = "Most unfilled shifts" });
+
+            filterSet.JobStatuses = new List<FilterField<JobStatuses>>
+                    {
+                        new FilterField<JobStatuses>() { Value = JobStatuses.New },
+                        new FilterField<JobStatuses>() { Value = JobStatuses.Open },
+                        new FilterField<JobStatuses>() { Value = JobStatuses.InProgress },
+                        new FilterField<JobStatuses>() { Value = JobStatuses.Done },
+                        new FilterField<JobStatuses>() { Value = JobStatuses.Cancelled },
+                    };
+
+
+
+            if (jobStatus != null)
+            {
+                filterSet.JobStatuses.Where(js => js.Value == jobStatus).First().IsSelected = true;
+            }
+            else
+            {
+                filterSet.JobStatuses.Where(js => js.Value == JobStatuses.New).First().IsSelected = true;
+                filterSet.JobStatuses.Where(js => js.Value == JobStatuses.Open).First().IsSelected = true;
+                filterSet.JobStatuses.Where(js => js.Value == JobStatuses.InProgress).First().IsSelected = true;
+            }
+
+            if (jobStatus == JobStatuses.Cancelled || jobStatus == JobStatuses.Done)
+            {
+                filterSet.OrderBy.Where(ob => ob.Value == OrderBy.DateStatusLastChanged_Descending).First().IsSelected = true;
+            }
+            else
+            {
+                filterSet.OrderBy.Where(ob => ob.Value == OrderBy.RequiringAdminAttention).First().IsSelected = true;
```

Comment:
We should make sure whatever default OrderBy values are on these lines are actually in our list of OrderBys.